### PR TITLE
fix: `guildKick` not working

### LINKED ISSUE #240: `Guild.kick()` does not work
**Slash command handler**
```dart
Future<void> kick(
    SlashCommandInteractionEvent event, GuildConfigDoc? configDoc) async {
  // args
  final userId = event.getArg('user').value as String;
  var reason = 'No reason';
  if (event.args.length > 1) {
    reason = event.getArg('reason').value as String;
  }
  // implementation
  final guild = await event.interaction.guild!.getOrDownload();

  final pms = event.interaction.memberAuthorPermissions!;
  if (pms.kickMembers) {
    await guild.kick(userId.toSnowflakeEntity(), auditReason: reason);
    await event.respond(normalEmbed('Kicked <@$userId> !'));
    await logActivity(
        event,
        configDoc!.botLogChan,
        '<@${event.interaction.userAuthor!.id}> '
        '**kicked** <@$userId> | Reason: **$reason**');
  } else {
    await event.respond(normalEmbed(noPermsString), hidden: true);
  }
}

/*
 * Args for command 
 */

final userArgKick = CommandOptionBuilder(
    CommandOptionType.user, 'user', 'Mention a user to kick',
    required: true);

final reasonArgKick = CommandOptionBuilder(
    CommandOptionType.string, 'reason', 'Type in a reason to kick');
```

**Context**
- Made sure that `configDoc` and `event.interaction.guild` is not null (hence, `event.interaction.memberAuthorPermissions` is also not null) in a previous step.
- Confirmed that `logActivity` does what it's supposed to.
- In testing environment, the bot had admin permission and the member that it was trying to kick did not have any permissions (or any roles)
- Given the above context, `onHttpError` did not fire when this command was invoked

**Desktop**
 - OS: Windows 11
 - Dart version [2.14.4]
 - Nyxx version [2.1.1]

### PR DESCRIPTION
**Pull request by 61soldiers**
# Description

Fixes #240 | `IHttpEndpoints.guildKick` not working due to a typo.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [ ] Ran `dartanalyzer --options analysis_options.yaml .`
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by HarryET** [comment]
Can you revert the formatting for now please and only fix the bug?

**Review by HarryET — CHANGES_REQUESTED** [review]
Remove Formatting. Just add bug fix.

**Review by HarryET — DISMISSED** [review]

File: lib/src/internal/_HttpEndpoints.dart [hunk 1]
```diff
@@ -1865,4 +1865,4 @@ class _HttpEndpoints implements IHttpEndpoints {
   @override
   String getRoleIconUrl(Snowflake roleId, String iconHash, String format, int size) =>
       "${Constants.cdnUrl}/role-icons/$roleId/$iconHash.$format?size=$size";
-}
+}
```
Comment (l7ssha) [inline lib/src/internal/_HttpEndpoints.dart:1868]: Removing trailing new line wasn't needed tho. That should stay in place

**Review by l7ssha — CHANGES_REQUESTED** [review]

File: lib/src/internal/_HttpEndpoints.dart (hunk 1 above)
Comment (61soldiers) [reply lib/src/internal/_HttpEndpoints.dart:1868]: https://github.com/nyxx-discord/nyxx/blob/1ea7a859168c6fcabfcfa4778fb1dd3d1f97c7cc/lib/src/internal/_HttpEndpoints.dart#L1865-L1868

There wasn't a trailing new line in the first place

Comment (l7ssha) [reply lib/src/internal/_HttpEndpoints.dart:1868]: there is but github doesnt show that, but its should lack of new line. Besides if new line wansnt present to begin with diff woudn't show that line as changed
image

Comment (61soldiers) [reply lib/src/internal/_HttpEndpoints.dart]: Didn't know that. Added trailing new line

**Review by l7ssha — APPROVED** [review]

File: lib/src/internal/_HttpEndpoints.dart (hunk 1 above)
Comment (l7ssha) [reply lib/src/internal/_HttpEndpoints.dart]: thanks

**Comment by l7ssha** [comment]
Thanks @61soldiers for contributing. Will merge and push `2.1.2` version today late night.

**Review by HarryET — APPROVED** [review]

**Pull request merged**